client/app/hosting/hosting.controller.js

```diff
@@ -520,7 +520,13 @@ angular
           })
           .then(() => {
             if (moment().isAfter(moment($scope.hostingProxy.lastOvhConfigScan).add(12, 'hours'))) {
-              return HostingOvhConfig.ovhConfigRefresh($stateParams.productId);
+              return HostingOvhConfig.ovhConfigRefresh($stateParams.productId, { returnErrorKey: '' }).then(data => data).catch((err) => {
```

Comment:
`.then(data => data)` is required?

According to the [API description](https://api.ovh.com/console/#/hosting/web/%7BserviceName%7D/ovhConfigRefresh#POST), I would suggest you to use: `hostingWebTask` which is the response type  instead of `data`.